src/stdlib/SDL_malloc.c

```diff
@@ -6352,6 +6352,17 @@ static struct
     real_malloc, real_calloc, real_realloc, real_free, { 0 }
 };
 
+// Define this if you want to track the number of allocations active
+// #define TRACK_ALLOCATION_COUNT
+#ifdef TRACK_ALLOCATION_COUNT
```

Comment:
Should the macro start with `SDL_`?

Reply:
Yes, since we’re documenting this that’s a good idea.